templates/.eslintrc.json

```diff
@@ -22,13 +22,10 @@
   "extends": [
     "airbnb",
     "plugin:jest/recommended",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "prettier",
```

Comment:
Seems like we don't really need this one?

https://github.com/prettier/eslint-plugin-prettier/blob/master/eslint-plugin-prettier.js#L110-L111